lib/code_corps/model/github_issue.ex

```diff
@@ -25,7 +25,7 @@ defmodule CodeCorps.GithubIssue do
     belongs_to :github_user, CodeCorps.GithubUser
 
     has_many :github_comments, CodeCorps.GithubComment
-    has_many :tasks, CodeCorps.Task
+    has_one :task, CodeCorps.Task
```

Comment:
Do we have an indication of everything that will need changed in our client? We're going to need to update them side-by-side.

Reply:
I was hoping you would have a better grasp, since you did some client work already on it.

My guess is, not much would change other than projectGithubRepo related stuff, since most of our UI is from the perspective of `Task` and `Comment`, not `GithubIssue` and `GithubComment`, and in that direction (task 1 - 1 issue), nothing has changed compared to the previous (task n - 1 issue)